packages/dom/src/serialize-cssom.js

```diff
@@ -35,6 +43,21 @@ export function serializeCSSOM({ dom, clone, resources, cache }) {
 
       cloneOwnerNode.parentNode.insertBefore(style, cloneOwnerNode.nextSibling);
       cloneOwnerNode.remove();
+    } else if (styleSheet instanceof window.CSSStyleSheet && styleSheet.href && styleSheet.href.startsWith('blob:')) {
```

Comment:
Why do we need `styleSheet instanceof window.CSSStyleSheet` check here? Isn't every stylesheet in `dom.styleSheets` of this type?